Microwave.Classes/Controllers/UserInterface.cs

```diff
@@ -79,6 +81,19 @@ public void OnTimePressed(object sender, EventArgs e)
                     time += 1;
                     myDisplay.ShowTime(time, 0);
                     break;
+                case States.COOKING:
+                    myCooker.AddCookingTime();
+                    break;
+            }
+        }
+
+        public void OnSubtractTimePressed(object sender, EventArgs e)
+        {
+            switch (myState)
+            {
+                case States.COOKING:
```

Comment:
Defining a default behavior when not in a cooking state would help on readability.

Reply:
Agree. But following code standard of existing code, that has no default state.